.travis.yml

```diff
@@ -38,3 +38,5 @@ notifications:
       - antonbachin@yahoo.com
     on_success: always
     on_failure: always
+
+sudo: required
```

Comment:
BTW, you also have the option to submit a PR, tell us not to review it, and use it to trigger CI in this repo :) Not sure why `sudo: required` would be required (heh). Perhaps the main Lwt repo needs it now too, but we are squeezing by due to some legacy support from Travis...?